### DISCUSSION TITLE
Is it possible to add a comment to an issue?

### DISCUSSION THREAD

**Discussion by gep13** [ubisoft/NGitLab#513] [Q&A] [answered]
I have had a look, but I don't see an obvious way of adding a comment to an issue on GitLab using NGitLab.

Have I missed something?

Thanks

**Comment by meziantou** [comment, marked as answer]
````c#
IGitLabClient client;
client.GetProjectIssueNoteClient(123).Create(new ProjectIssueNoteCreate
{
    IssueId = 1,
    Body = "sample",
});
````

**Reply by gep13** [reply]
That looks like exactly what I would need! Thank you, I will take this for a spin.

**Reply by gep13** [reply]
@meziantou just wanted to circle back to say that this does exactly what I want! Thanks again for showing me this! Not sure how I missed this, as I thought I went through all the available clients, and didn't see anything that I thought would be relevant.
